packages/components/src/templates/expandable.stories.ts

```diff
@@ -0,0 +1,110 @@
+          </div>
+        </sd-expandable>
+      </div>
+      <div class="background-sample bg-neutral-100 p-4">
+        <sd-expandable
+          style="--gradient-color-start: rgba(246, 246, 246, 0);--gradient-color-end: rgba(246, 246, 246, 1)"
+        >
+          <div class="sd-paragraph h-full">
+            For us, diversity equals strength. By taking the views and experiences of a wide range of people into
+            account, we provide room for creative solutions and ensure that we are equipped for the future. A diverse
+            and inclusive working environment is important to us. We benefit from the perspectives of different genders,
+            age groups, educational levels and backgrounds, thus guaranteeing that we are successful together.
+          </div>
+        </sd-expandable>
+      </div>
+      <div class="background-sample bg-primary-100 p-4">
+        <sd-expandable
+          style="--gradient-color-start: rgba(236, 240, 249, 0);--gradient-color-end: rgba(236, 240, 249, 1)"
+        >
+          <div class="sd-paragraph h-full">
+            For us, diversity equals strength. By taking the views and experiences of a wide range of people into
+            account, we provide room for creative solutions and ensure that we are equipped for the future. A diverse
+            and inclusive working environment is important to us. We benefit from the perspectives of different genders,
+            age groups, educational levels and backgrounds, thus guaranteeing that we are successful together.
+          </div>
+        </sd-expandable>
+      </div>
+      <div class="background-sample bg-primary p-4">
+        <sd-expandable inverted>
+          <div class="sd-paragraph h-full text-white">
+            For us, diversity equals strength. By taking the views and experiences of a wide range of people into
+            account, we provide room for creative solutions and ensure that we are equipped for the future. A diverse
+            and inclusive working environment is important to us. We benefit from the perspectives of different genders,
+            age groups, educational levels and backgrounds, thus guaranteeing that we are successful together.
+          </div>
+        </sd-expandable>
+      </div>
+    </div> `
+};
+
+/**
+ * - `leadtext` (default): use the class `sd-leadtext`
+ * - `paragraph`: use the class `sd-paragraph`
+ */
+
+export const FontStyleOptions = {
+  name: 'Font style options',
+  render: () =>
+    html`<div class="flex flex-col gap-4">
+      <div class="background-sample bg-neutral-100 p-4">
+        <sd-expandable
+          style="--gradient-color-start: rgba(246, 246, 246, 0);--gradient-color-end: rgba(246, 246, 246, 1)"
+        >
+          <div class="sd-leadtext">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
```

Comment:
Global economic growth continues to be an important support factor for the capital markets. The US economy in particular is running like a well-oiled machine. This is unlikely to change much in the coming quarters. Our economists expect real economic output to increase by 2.7 per cent in 2024 as a whole. Unlike in previous months, they also expect the economic outlook to improve in other regions, above all in the eurozone. Here, gross domestic product (GDP) is expected to grow by 0.6 per cent over the course of the year. Germany brings up the rear with GDP growth forecast to be only slightly above zero. As the economies of China and Japan are stabilising at the same time, the global economy is expected to grow by 2.9% in 2024, slightly more than in the previous year. The growth gap between the most important regions is therefore closing somewhat.

**US inflation soon to reach its cyclical low**

Our judgement on inflation is mixed. Although the global trend of falling inflation is continuing, it is slowing down. In the US, the cyclical low in core inflation, i.e. the price trend excluding energy and food, is likely to be reached in the summer. However, inflation remains above the US Federal Reserve's inflation target. If inflation in the United States does not fall any further, this will generally support interest rates on the bond market. We therefore do not expect yields on the bond market to fall, especially for bonds with longer maturities. By contrast, the eurozone is later in the economic cycle and is structurally not as fast-growing as the US, which is why the European Central Bank (ECB) can expect a further decline in price increases. Inflation is likely to approach the ECB's target of two per cent by mid-2025.